### PR TITLE
adding query params to /api/v1/stores/{storeId}/invoices to filter data

### PR DESCRIPTION
**Pull request by SakerOmera**
…ers to limit results

### CODE REVIEW

**Comment by SakerOmera** [comment]
create /api/v2/stores/{storeId}/invoices for customized query parameters to limit results

**Comment by NicolasDorier** [comment]
concept ACK, but maybe inside the same current route? ping @Kukks

**Comment by Kukks** [comment]
Yes same route not v2. We should also standardize a paging model for all list results

**Comment by SakerOmera** [comment]
Thanks @Kukks @NicolasDorier for the feedback. 
Completed necessary changes to allow **/v1/invoices** to accept more query parameters to filter data.

Can you please take a look and let me know if you have any comments?

**Comment by NicolasDorier** [comment]
@SakerOmera can you rebase? @pavlenex we can add that for next milestone I think.

**Comment by SakerOmera** [comment]
@NicolasDorier rebased and all tests passed. Can you please share any timeline when this milestone is gonna be live/available?

**Comment by pavlenex** [comment]
@SakerOmera Next release approx in 2 weeks or less.

File: BTCPayServer.Client/BTCPayServerClient.Invoices.cs [hunk 1]
```diff
@@ -9,13 +9,43 @@ namespace BTCPayServer.Client
...
-        public virtual async Task<IEnumerable<InvoiceData>> GetInvoices(string storeId, bool includeArchived = false,
+        // public virtual async Task<IEnumerable<InvoiceData>> GetInvoices(string storeId, bool includeArchived = false,
+        //     CancellationToken token = default)
+        // {
+        //     var response =
+        //         await _httpClient.SendAsync(
+        //             CreateHttpRequest($"api/v1/stores/{storeId}/invoices",
+        //                 new Dictionary<string, object>() {{nameof(includeArchived), includeArchived}}), token);
+        //     return await HandleResponse<IEnumerable<InvoiceData>>(response);
+        // }
```
Comment (NicolasDorier) [inline BTCPayServer.Client/BTCPayServerClient.Invoices.cs:20]: remove this comment

**Comment by NicolasDorier** [comment]
Please remove all the formatting change you did, this make it difficult to review.

File: BTCPayServer.Client/BTCPayServerClient.Invoices.cs (hunk 1 above)
Comment (SakerOmera) [reply BTCPayServer.Client/BTCPayServerClient.Invoices.cs]: @NicolasDorier Done

**Comment by SakerOmera** [comment]
@NicolasDorier PR is updated, removed content formatting.

**Comment by Kukks** [comment]
Replaced by #2461?

**Pull request closed without merging**